ide/src/extension.ts

```diff
@@ -39,6 +43,11 @@ export async function activate(context: vscode.ExtensionContext) {
       );
       context.subscriptions.push(disposable);
     });
+
+    focus_status_bar_item = vscode.window.createStatusBarItem();
```

Comment:
Since the scope of the status bar (as I understand) is to also indicate when Flowistry is computing/executing something, I needed to define this outside just the scope of the `FocusMode` class (same as global state comment).